src/main/kotlin/com/chattriggers/ctjs/commands/Command.kt

```diff
@@ -1,119 +1,51 @@
 package com.chattriggers.ctjs.commands
 
-import com.chattriggers.ctjs.CTJS
 import com.chattriggers.ctjs.triggers.Trigger
-import com.chattriggers.ctjs.utils.kotlin.asMixin
-//#if MC==10809
-import com.chattriggers.ctjs.launch.mixins.transformers.CommandHandlerAccessor
-import net.minecraft.command.CommandBase
-import net.minecraft.command.CommandException
-import net.minecraft.command.ICommandSender
-import net.minecraft.util.BlockPos
-import net.minecraftforge.client.ClientCommandHandler
-//#else
-//$$ import com.mojang.brigadier.CommandDispatcher
-//$$ import com.mojang.brigadier.arguments.StringArgumentType
-//$$ import com.mojang.brigadier.context.CommandContext
-//$$ import net.minecraft.commands.CommandSourceStack
-//$$ import net.minecraft.commands.Commands
-//$$ import com.mojang.brigadier.tree.CommandNode
-//#endif
+import gg.essential.api.EssentialAPI
+import gg.essential.api.commands.DefaultHandler
+import gg.essential.api.commands.Greedy
+import gg.essential.api.commands.Command as EssentialCommand
 
+// TODO(BREAKING): Removed callback parameter since nested subcommands are very difficult to implement,
+// you have to use a subcommand and then @Options
```

Comment:
I suggested they make a dsl option for commands a couple of months back, but there wasn't any reaction to it.  I could possibly try to do hacky stuff by adding annotations at runtime somehow but even then, I don't think it would work.

Reply:
Well it is what it is, we can see if it's possible later.